src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -113,39 +97,42 @@ const SulPeopleTableView = ({items, hasHeading}: Props) => {
             <legend className="sr-only">Filter by speciality</legend>
-            <label className="group hidden cursor-pointer border-r-0 md:block">
+            <label className="group cursor-pointer border-r-0 md:block">
               <input
-                type="checkbox"
+                type="radio"
+                name="specialist"
                 className="peer sr-only"
-                checked={!typeFilter.length}
-                onChange={() => updateTypeFilter()}
+                checked={!typeFilter}
+                onChange={() => setTypeFilter("")}
               />
-              <span className="block rounded-l-full border border-r-0 border-black-80 p-9 px-20 text-18 no-underline group-hover:border-cardinal-red-dark group-hover:text-cardinal-red-dark group-hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10">
-                All specialists
+              <span className="block whitespace-nowrap rounded-l-full border border-r-0 border-black-80 p-4 px-16 text-14 leading-[30px] no-underline group-hover:border-cardinal-red-dark group-hover:text-cardinal-red-dark group-hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-hover:bg-[#979694] peer-hover:bg-opacity-10 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10 peer-active:bg-[#979694] peer-active:bg-opacity-10 md:text-16">
+                All
               </span>
             </label>
             <label className="group cursor-pointer">
               <input
-                type="checkbox"
+                type="radio"
+                name="specialist"
                 className="peer sr-only"
-                checked={typeFilter.includes("subject specialist")}
-                onChange={() => updateTypeFilter("subject specialist")}
+                checked={typeFilter === "subject specialist"}
+                onChange={() => setTypeFilter("subject specialist")}
               />
-              <span className="block items-center rounded-l-full border border-r-0 border-black-80 p-9 pr-20 text-18 no-underline group-hover:border-cardinal-red-dark group-hover:text-cardinal-red-dark group-hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10 md:rounded-l-none">
+              <span className="block items-center whitespace-nowrap border border-r-0 border-black-80 p-4 px-16 text-14 leading-[30px] no-underline group-hover:border-cardinal-red-dark group-hover:text-cardinal-red-dark group-hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-hover:bg-[#979694] peer-hover:bg-opacity-10 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10 peer-active:bg-[#979694] peer-active:bg-opacity-10 md:rounded-l-none md:text-16">
```

Comment:
As I mentioned, you need `peer-checked`. `peer-active` is only when the user is actively engaged on the element. as soon as they click/focus off the element, those styles are lost. Also, to my knowledge, a simple background color indicator as this is not sufficient for accessibility. The gray does not have a high enough contrast with the white to indicate its selected state.

Reply:
Thank you. I see what you mean.  They are going to make some changes to these toggles so I'll bring up the accessibility part and make those changes in this PR, unless you have concerns about that.

Reply:
Also FWIW, the contrast needs to be 3:1